delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/SearchParams.scala

```diff
@@ -0,0 +1,25 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.model
+
+import org.apache.jena.iri.IRI
+
+/**
+  * Search parameters to filter resources.
+  *
+  * @param deprecated the optional deprecation status of the resources
+  * @param rev        the optional revision of the resources
+  * @param createdBy  the optional identity id who created the resource
+  * @param updatedBy  the optional identity id who updated the resource
+  * @param types      the optional types of the resources
+  */
+final case class SearchParams(
```

Comment:
missing field `schema: Option[ResourceRef]`

For now this is alright, but soon we'll need to subclass it, since we need `SearchParams` which can contain organization and project labels

Reply:
This was directly ported from the service, but yeah, we need to add all other possible filters and stay consistent even if they're not applicable to realms.

Reply:
I'm adding the schemas for now, but we need to reconsider the search params structure; I think we need an ADT because some of these options don't make sense together.